Kernel/Jail.cpp

```diff
@@ -19,16 +19,19 @@ static JailIndex generate_jail_id()
     return s_jail_id.fetch_add(1);
 }
 
-NonnullRefPtr<ProcessList> Jail::process_list()
+RefPtr<ProcessList> Jail::process_list()
 {
     return m_process_list;
 }
 
-ErrorOr<NonnullRefPtr<Jail>> Jail::create(NonnullOwnPtr<KString> name)
+ErrorOr<NonnullRefPtr<Jail>> Jail::create(NonnullOwnPtr<KString> name, EnforcePIDIsolation enforce_pid_isolation)
```

Comment:
Can't we accept the flags here as well? This tends to grow in number of arguments as soon as we think of other isolation options (networking, filesystem, etc.). I can imagine the invoker would translate the `::Default` flag to the individual default isolation option flags and passing those to this method.